scripts/evaluation_script.py

```diff
@@ -48,3 +53,71 @@
     pic_loc="output/",
     pic_name="weekly_predicted_uptake.png",
 )
+
+# evaluate time-varying MSPE #
+date_mspe = pl.DataFrame()
+
+for i in range(len(rt_pred_2023)):
```

Comment:
rework as a list comprehension:
```python
date_mspe = pl.concat([
  get_mspe(whatever)
  for whatever in so_and_so
])
```

Reply:
Addressed in [5f2e366](https://github.com/CDCgov/cfa-immunization-uptake-projection/pull/37/commits/5f2e366c5115d1e769b602f4aacb4d854cce534e)